travis.sh

```diff
@@ -11,7 +11,7 @@
 
 export MOVEIT_CI_DIR=$(dirname ${BASH_SOURCE:-$0})  # path to the directory running the current script
 export REPOSITORY_NAME=$(basename $PWD) # name of repository, travis originally checked out
-export ROS_WS=${ROS_WS:-/root/ws_moveit} # location of ROS workspace
+export ROS_WS=${ROS_WS:-/root/ros_ws} # location of ROS workspace
```

Comment:
@rhaschke and @ruffsl
We really need to sync the location of the ros workspace across moveit2, moveit_ci and elsewhere. 
I don't have a strong opinion one way or the other but let's take the time to agree so that we don't deviate

Reply:
Context: https://github.com/ros-planning/moveit2/pull/44#discussion_r267158151

Reply:
I think a nice convention is `ROS_WS` env for the layered workspace that includes upstream dependencies, and something like `MOVEIT2_WS` for the workspace with moveit2! packages.

Reply:
I think, we consistently use ROS_WS in docker and moveit_ci to refer to the workspace location.
Why do you see the need to distinguish ROS_WS and MOVEIT_WS?

Reply:
I've always advocated putting the ``ws_`` in front, so the workspaces are ordered together within the parent folder. ``ws_moveit`` is the standard we've been following for a few years now: https://moveit.ros.org/install/source/